src/pages/index.mdx

```diff
@@ -4,23 +4,69 @@ import { SiteLayout } from "../components/layout/SiteLayout";
 
 <SiteLayout>
 
-<h1 className="text-center font-display">11th Annual oSTEM Conference 2021</h1>
-
-<ConferenceCountdown />
-
-<img src="/images/2021/podium-presenters.jpg" alt="" className="rounded-3xl p-3"/>
-
-<p className="text-center text-3xl font-display">Your seat at our table</p>
-<p className="text-center text-3xl font-display">
-  October 28 &ndash; 31, 2021, Virtual
-</p>
-
-<div className="text-2xl flex justify-center p-8">
-  <CTAButton onClick="/2021/rfp">Submit a Proposal</CTAButton>
+<div className="border-b-4 border-mulberry border-solid pb-4">
+    <p className="flex text-center text-5xl font-display font-black text-mulberry">11th Annual oSTEM Conference 2021</p>
+    <p className="flex justify-center text-2xl font-display italic">Your seat at our table</p>
+    <hr/>
+    <p className="flex justify-center text-3xl font-display">
+    October 28 &ndash; 31, 2021 | Virtual
+    </p>
+</div>
+<div class="grid grid-cols-5 gap-y-2 mt-9">
```

Comment:
I did check it in mobile and I thought it looked fine since the main subject of the image was still in focus and I didn't think the image scaling was too bad. The text was still readable and not too compressed.

Reply:
Added breakpoints for the columns. I probably want to add similar size-constraints for the image tinting, but that'll be for the future.